src/main/java/com/cloudbees/jenkins/support/impl/HeapUsageHistogram.java

```diff
@@ -33,6 +33,10 @@
     // first 200 classes so 203 lines required because of the header
     private static final int MAX = 203;
 
+    // disabled by default because of JENKINS-49931
+    // to be reviewed in the future.
+    private static /*final*/ boolean DISABLED = Boolean.parseBoolean(System.getProperty(HeapUsageHistogram.class.getCanonicalName() + ".DISABLED", "true"));
```

Comment:
BTW use of the negative sense is discouraged. Rather use `ENABLED` and default to false.

Also use `SystemProperties` since we are on Jenkins 2.x now.

Reply:
@jglick thanks! I will use ENABLE instead of DISABLE in the future. I have created the https://issues.jenkins-ci.org/browse/JENKINS-50010 in order to let admins enable/disable components through UI, and also a common way to enforce the disable of components as a system property.